oqs_scripts/testrun.py

```diff
@@ -0,0 +1,38 @@
+import json
+import sys
+import subprocess
+import os
+import time
+
+# Parameter checks already done in shellscript
+
+with open("assignments.json", "r") as f:
+   jsoncontents = f.read();
+
+onlysigoutput = False
+if len(sys.argv)>1:
+   onlysigoutput=True
+
+assignments = json.loads(jsoncontents)
+for sig in assignments:
+    print("Testing %s:" % (sig))
+    for kem in assignments[sig]:
+       # assemble testing command
+       cmd = "(echo \'GET /\'; sleep 0.2) | ../build/tool/bssl client -connect test.openquantumsafe.org:"+str(assignments[sig][kem]) + " -root-certs CA.crt 2>&1"
+       if kem!="*": # don't prescribe KEM
+          cmd=cmd+" -curves "+kem
+       output = os.popen(cmd).read()
+       if not ("Successfully" in output):
+           print("Error with command '%s': \n%s\n" % (cmd, output))
+           if (not onlysigoutput):
```

Comment:
We can probably remove this flag, and just print something like `kem+sig failed.` whenever there's an error, and nothing otherwise. In fact I think it would be better if we just hooked this whole thing up to `pytest`, which would let us parallelize this.

Reply:
Not doing this as part of regular testing as per discussion below. Suggest to close this point.